### PR TITLE
Add 'uppercase' and 'lowercase' to script editor

### PR DESCRIPTION
**Pull request by sersoong**
Easy to convert case in 2.1.
like 3.0.
test

### CODE REVIEW

**Comment by sersoong** [comment]
I remake this.
Optimization of its implementation.
Maybe you can review it again.
@akien-mga

File: scene/gui/text_edit.cpp [hunk 1]
```diff
@@ -3370,6 +3370,21 @@ void TextEdit::cut() {
...
 }
 
+void TextEdit::convert_case(const String &p_Case) {
+	if (selection.active) {
+		String text = _base_get_text(selection.from_line, selection.from_column, selection.to_line, selection.to_column);
+		selection.active = false;
+		_remove_text(selection.from_line, selection.from_column, selection.to_line, selection.to_column);
+		cursor_set_line(selection.from_line);
+		cursor_set_column(selection.from_column);
+		if (p_Case == "UPPER") {
```
Comment (akien-mga) [inline scene/gui/text_edit.cpp]: If you want to do it this way, it should be an enum, not a magic string.

Comment (sersoong) [reply scene/gui/text_edit.cpp]: :p

**Comment by sersoong** [comment]
Changed to enum.

**Pull request merged**